src/compiler.rs

```diff
@@ -324,8 +324,14 @@ impl AmberCompiler {
 
     #[cfg(not(windows))]
     fn find_bash() -> Option<Command> {
-        let mut command = Command::new("/usr/bin/env");
-        command.arg("bash");
-        Some(command)
+        if env::var("GITHUB_ACTIONS_BASH_CONTAINER").is_ok() {
+            let mut command = Command::new("/usr/bin/docker");
+            command.args(["exec", "--workdir", "/root", "--user", "405", "bash", "bash"]);
```

Comment:
Not sure why you're repeating the `bash` argument; is this necessary?

Reply:
The first bash is the name of the running container, and the second bash is an executable. I will rename it to a meaningful one.

Reply:
I've renamed the container from `bash` to `test_container`.